### ISSUE TITLE
Migrate Cassandra keyspace to NetworkTopologyStrategy

### ISSUE THREAD

**Issue by thefosk** [Mashape/kong#350]
We should change the replication strategy of the Kong keyspace to [`NetworkTopologyStrategy `](http://docs.datastax.com/en/cassandra/2.0/cassandra/architecture/architectureDataDistributeReplication_c.html?scroll=concept_ds_yt4_m4f_fk__nts) as opposed to having `SimpleStrategy` to make it easier to scale a Cassandra cluster. From the Cassandra documentation:

> NetworkTopologyStrategy: Highly recommended for most deployments because it is much easier to expand to multiple data centers when required by future expansion.

It means that `NetworkTopologyStrategy` can work pretty much in every scenario (including single-machine, or single-datacenter), while `SimpleStrategy` is more limited.

Doing this should be as easy as executing:

```sql
ALTER KEYSPACE kong WITH REPLICATION = {'class' : 'NetworkTopologyStrategy', 'DC1' : 1};
```

This also brings another question: in a production environment I might be wanting to setup my own Cassandra cluster with some very specific settings, how do we prevent Kong migrations from being executed and reverting these settings?

**Comment by thefosk** [comment]
This seems to work when invoked by `cqlsh`, but the same command fails when executed from the migration script:

```
[ERR] Cannot record migration 2015-06-09-170921_0.4.0: Cassandra error: Cassandra returned error (Unavailable exception): "Cannot achieve consistency level ONE"
```

Very weird since the same command seems to be working manually.

**Comment by thibaultCha** [comment]
See #543 for what needs to be done here.

**Comment by thibaultCha** [comment]
Implemented with #634.
